jdaviz/configs/imviz/plugins/aper_phot_simple/aper_phot_simple.vue

```diff
@@ -152,14 +151,21 @@
       </v-row>
     </div>
 
-    <div v-if="result_available">
-      <j-plugin-section-header>Photometry Results</j-plugin-section-header>
+    <v-row v-if="plot_available">
+      <!-- NOTE: the internal bqplot widget defaults to 480 pixels, so if choosing something else,
+           we will likely need to override that with custom CSS rules in order to avoid the initial
+           rendering of the plot from overlapping with content below -->
+      <jupyter-widget :widget="radial_plot" style="width: 100%; height: 480px" />
+    </v-row>
+
+    <div v-if="plot_available && fit_radial_profile">
+      <j-plugin-section-header>Gaussian Fit Results</j-plugin-section-header>

```

Comment:
@larrybradley specifically wanted the fit results to be on top, so moving this down is a step backwards. See https://github.com/spacetelescope/jdaviz/pull/1409#issuecomment-1170358197

Reply:
ok, thanks for that context.  This does keep them with the plot, but moves the plot _and_ the gaussian fit below the photometry results.  If we want to move them both back up to the top, then we might need to look into a javascript solution for detecting when the plot is rendered and show a spinner in the meantime, since the blank plot just suggests something isn't working.

Reply:
Moving things around need PO's blessing, I think. I am neutral on whatever that works.

The warning itself pops up as advertised though it is hard for me to judge whether 5000 is a good cutoff or not.

Reply:
Maybe this is a Jenn question, but would you consider having two buttons? calculate photometry and plot curve or something like that, so the user can decide not to show either if they do not need it and they would see first what they want.

Reply:
I like that idea, and then the warning is more tied to just the plotting step rather than computing the photometry itself - would you like me to do that as part of this PR?

Reply:
I originally wanted the plotting to be a separate plugin...

I think adding another button would introduce too much diff that goes beyond the scope of this PR. Is there a way to do the warning without moving things around?

Reply:
Yes, I can easily revert changing the order but keep the warning, but see my concerns above for why I think this isn't ideal.  But if the plan is to follow-up (relatively soon) with splitting into two buttons/sections, then the warning on its own is still an incremental improvement.

I'm also curious what @larrybradley thinks - was your original request from the comment @pllim linked just to keep the gaussian results tied closely with the plots, or to have the plot and gaussian results _before_ the photometry results?

Reply:
Stick with the warning for this PR and open a new one (if you have time now or an issue if you don't) for splitting the button, please.

Reply:
I did revert the re-ordering for this PR.  I'm working on some smarter thresholds for the warning and will then open a follow-up ticket for either considering re-ordering, splitting the buttons, or splitting the plugin.